src/main/java/io/confluent/connect/jdbc/sink/RecordValidator.java

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright 2018 Confluent Inc.
```

Comment:
```suggestion
 * Copyright 2020 Confluent Inc.
```

Reply:
This is best fixed in the pom with a year bump, and probably noisy enough to push to a different PR.

Reply:
TIL: you can have different copyright dates for each file, as long as it's the same as or more recent than the project's inception year.

IIRC, we actually don't want to change all of the files at once.